### PR TITLE
feat(ui): set active device when viewing details

### PR DESCRIPTION
**Pull request by Caleb-Ellis**
## Done

- Set active device when viewing device details

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to /MAAS/r/devices
- Click on a device, and check that a loading spinner shows in the header initially before disappearing
- Check in Redux DevTools that the device was successfully set as active

## Fixes

Fixes canonical-web-and-design/app-tribe#558

### CODE REVIEW

File: ui/src/app/devices/components/DeviceHeaderForms/AddDeviceForm/AddDeviceInterfaces/AddDeviceInterfaces.tsx [hunk 1]
```diff
@@ -94,6 +94,7 @@ export const AddDeviceInterfaces = (): JSX.Element => {
                 className: "ip-assignment-col",
                 content: (
                   <IpAssignmentSelect
+                    label={null}
```
Comment (Caleb-Ellis) [inline ui/src/app/devices/components/DeviceHeaderForms/AddDeviceForm/AddDeviceInterfaces/AddDeviceInterfaces.tsx:97]: driveby fix of the add device form

**Review by huwshimi — APPROVED** [review]
